components/buildList/js/buildList.controller.js

```diff
@@ -237,4 +237,242 @@ buildApp.controller('buildListController', ['$scope', '$q', '$interval', 'buildL
 	}
 
 	init();
+}]).controller('buildListController', ['$scope', '$q', '$interval', 'buildListService', function ($scope, $q, $interval, buildListService) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function has 239 lines, which is greater than the 100 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3AS138)